lib/kubeclient/common.rb

```diff
@@ -51,6 +53,9 @@ def initialize_client(
       validate_auth_options(auth_options)
       handle_uri(uri, path)
 
+      @class_owner = class_owner
+      @entities = {}
+      @discovered = false
```

Comment:
That is done in handle uri (same like @api_endpoint) so I now always init it there.
also renamed group to api_group (like existing api_version) and changed code to rely on initialization.